### DISCUSSION TITLE
Thanks and a few gw observations

### DISCUSSION THREAD

**Discussion by kamold** [jilleb/MQB-FPA#76] [Q&A]
Firstly, wanted to thank you for this amazing project!
Have sent some thanks as sponsorship.

Have done DCC slider mod on a Mk3 Superb (changed gw from 5Q0 to 3Q0 C, installed 4388 sw).

Also did the same in a B8 Passat.  It had 3Q0 gw from factory.  Installed AC software however this gave an error in rear radars (can't communicate with mirrors 123063 1192035 databus missing message gateway_76 spiegel_01) and I discovered that AC sw 5388 and above only supports newer 2Q0 radars.
Installed AA software and applied datasets all happy now.
Hopefully this helps someone.

**Comment by jjff2** [comment]
No. Flash ur radar to higher version. 3q0 index G is fine. Then do calibration.

**Comment by Mex77** [comment]
Will be trying to do this on my Passat Alltrack B8 220tsi 2017.
But alot of people say that 3Q0907530A* Conti gateway doesn't work well with 3Q0 bcm (I have 5Q0) and rear radars?

**Reply by kamold** [reply]
AA software works fine with Passat rear radars.
Newer than that you need to also update rear radars and perform calibration as pointed out above.

**Reply by Mex77** [reply]
Ok, thanks..
Don't want to end up with that fan calibration thingy..

**Comment by Mex77** [comment]
Thank you..
